tests/framework/results/helpers.py

```diff
@@ -74,14 +74,15 @@ def columns_created(self) -> list[str]:
     def setup(self, builder: Builder) -> None:
         self.grade = builder.value.register_value_producer(
             "grade",
-            source=lambda index: self.population_view.get(index, "exam_score").map(
+            source=lambda index: self.population_view.get_attributes(index, "exam_score").map(
```

Comment:
It's extremely unclear to me whether or not these helpers should be calling get_attributes or get_private_columns. Might need to discuss this